complete_streaming_function.py

```
@@ -0,0 +1,394 @@
+async def linear_simulation_chat_stream(
+    request: SimulationChatRequest,
```

Ruff (0.14.1)

Undefined name `SimulationChatRequest`

(F821)